src/settings-ui/Settings.UI/Views/PowerAccentPage.xaml

```diff
@@ -90,7 +90,20 @@
                         </controls:Setting.ActionContent>
                     </controls:Setting>
                 </controls:SettingsGroup>
-                
+
+                <controls:SettingsGroup x:Uid="QuickAccent_Description" IsEnabled="{x:Bind Mode=OneWay, Path=ViewModel.IsEnabled}">
+                    <controls:Setting x:Uid="QuickAccent_Description_Indicator" Icon="&#xE946;">
```

Comment:
Let's move to the existing "behavior" group. No need to create a new group for only one setting if we have a good existing one.